lib/linguist/languages.yml

```diff
@@ -4856,13 +4856,23 @@ Volt:
   codemirror_mime_type: text/x-d
   language_id: 390
 Vue:
-  type: markup
+  type: programming
   color: "#2c3e50"
   extensions:
   - ".vue"
-  tm_scope: text.html.vue
+  tm_scope: source.vue
   ace_mode: html
+  codemirror_mode: vue
+  codemirror_mime_type: text/html
```

Comment:
The Travis build is failing because this reference cannot be found in CodeMirror. I think you want [`text/x-vue`](https://github.com/codemirror/CodeMirror/blob/ab8bf8e5efd210568893824ee7c8f8b08000906c/mode/vue/vue.js#L76).

Reply:
Thanks! Was busy with other things and didn't notice Travis was failing.